src/AppBundle/EventListener/CheckSQLiteEventSubscriber.php

```diff
@@ -57,4 +61,16 @@ public function handleDatabaseExceptions(ConsoleExceptionEvent $event)
             }
         }
     }
+
+    /**
+     * This method is triggered when kernel exception occurs. And checks if sqlite extension is enabled.
+     *
+     * @param GetResponseForExceptionEvent $event
+     */
+    public function onKernelException(GetResponseForExceptionEvent $event)
+    {
+        if (!extension_loaded('sqlite3')) {
```

Comment:
Well, any exception thrown during a Twig template rendering is wrapped in a Twig_Error_Runtime. You can check the chain of previous exceptions